uniffi_bindgen/src/interface/mod.rs

```diff
@@ -241,6 +241,42 @@ impl<'ci> ComponentInterface {
         }
     }
 
+    /// Identify if we are using an unsigned type in kotlin
```

Comment:
Nit: there's nothing Kotlin-specific about it at this level, so the "in kotlin" here risks confusion. (Explaining that we need this for the Kotlin bindings makes sense, but it does the same thing regardless of which bindings are being generated).